### ISSUE TITLE
GridActionsCellItem usage in v5 documentation examples do not work

### ISSUE THREAD

**Issue by TomDoesbRUG** [mui-org/material-ui-x#2614]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Attempted import error: 'GridActionsCellItem' is not exported from '@mui/x-data-grid'.

### Expected behavior 🤔

@mui/x-data-grid should export the `GridActionsCellItem`

### Steps to reproduce 🕹

Steps:

1. Create a column definition with type `actions`
2. Implement get actions using the GridActionsCellItem
3. Import the element from @mui/x-data-grid
4.

https://codesandbox.io/s/svzs3?file=/demo.tsx

### Context 🔦

Working with the new v5 mui/material i came across the new `action` column type: https://next.material-ui.com/components/data-grid/columns/#column-types

Since its working in the example I expected it to be implemented. However it doesnt seem to be available. Is there a (alpha/beta) release available where this works? 

### Your environment 🌎

<details>
  <summary>`npx @mui/envinfo`</summary>

```
  System:
    OS: macOS 11.4
  Binaries:
    Node: 15.13.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 7.7.6 - /usr/local/bin/npm
  Browsers:
    Chrome: 93.0.4577.82
    Edge: Not Found
    Firefox: 87.0
    Safari: 14.1.1
  npmPackages:
    @emotion/react: ^11.4.1 => 11.4.1 
    @emotion/styled: ^11.3.0 => 11.3.0 
    @mui/core:  5.0.0-alpha.46 
    @mui/icons-material: ^5.0.0-rc.0 => 5.0.0-rc.0 
    @mui/lab: 5.0.0-alpha.46 => 5.0.0-alpha.46 
    @mui/material: ^5.0.0-rc.0 => 5.0.0-rc.0 
    @mui/private-theming:  5.0.0-rc.0 
    @mui/styled-engine:  5.0.0-rc.1 
    @mui/styles: ^5.0.0-rc.0 => 5.0.0-rc.0 
    @mui/system:  5.0.0-rc.1 
    @mui/types:  7.0.0-rc.0 
    @mui/utils:  5.0.0-rc.1 
    @mui/x-data-grid: ^4.0.0 => 4.0.0 
    @types/react: ^17.0.16 => 17.0.16 
    react: ^17.0.2 => 17.0.2 
    react-dom: ^17.0.2 => 17.0.2 
    styled-components:  4.4.1 
    typescript: ^4.3.5 => 4.3.5 

```
</details>

**Comment by arthurix** [comment]
I'm having the same issue trying to use column type `actions`. I cannot import `GridActionsCellItem` as described in the example https://next.material-ui.com/components/data-grid/columns/#column-types. Maybe it was not merged in this version `4.0.0`.

##### Issue:
```Uncaught SyntaxError: The requested module '/@fs/.../@mui_x-data-grid.js?v=7f09afc9' does not provide an export named 'GridActionsCellItem'```

<details>
<summary>npx @mui/envinfo</summary>
<pre>
<code>
  System:
    OS: macOS 10.15.7
  Binaries:
    Node: 16.6.1 - ~/.nvm/versions/node/v16.6.1/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.20.3 - ~/.nvm/versions/node/v16.6.1/bin/npm
  Browsers:
    Chrome: 93.0.4577.82
    Edge: Not Found
    Firefox: 89.0
    Safari: 14.1.1
  npmPackages:
    @emotion/react: latest => 11.4.1
    @emotion/styled: latest => 11.3.0
    @mui/core:  5.0.0-alpha.46
    @mui/envinfo: ^2.0.0 => 2.0.0
    @mui/icons-material: ^5.0.0-rc.1 => 5.0.0-rc.1
    @mui/material: next => 5.0.0-rc.1
    @mui/private-theming:  5.0.0-rc.1
    @mui/styled-engine:  5.0.0-rc.1
    @mui/system:  5.0.0-rc.1
    @mui/types:  7.0.0-rc.1
    @mui/utils:  5.0.0-rc.1
    @mui/x-data-grid: ^4.0.0 => 4.0.0
    @types/react:  17.0.21
    react: ^17.0.1 => 17.0.2
    react-dom: ^17.0.1 => 17.0.2
</pre>
</code>
</details>

**Comment by m4theushw** [comment]
The `actions` column type was not released yet. We plan to release it tomorrow. The docs at https://next.material-ui.com/components/data-grid/columns/#column-types is pointing to the unreleased version. You can see which are the available column types in the **last stable release** by accessing https://material-ui.com/components/data-grid/columns/#column-types
